tool/site/client/src/components/dose-to-text/doseToTextForm.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react"
+import {Button, Form, Fieldset, Textarea} from "nhsuk-react-components"
+import {Field, Formik} from "formik"
+import ButtonList from "../common/buttonList"
+import BackButton from "../common/backButton"
+
+interface DoseToTextFormProps {
+  initialValues?: DoseToTextFormValues
+  onSubmit: (values: DoseToTextFormValues) => void
+}
+
+export interface DoseToTextFormValues {
+  doseToTextRequest: string
+}
+
+const DoseToTextForm: React.FC<DoseToTextFormProps> = ({
+  initialValues,
+  onSubmit
+}) => {
+  return (
+    <Formik<DoseToTextFormValues> initialValues={initialValues} onSubmit={values => onSubmit(values)}>
```

Comment:
could be wrong but I don't think you need the arrow here, could maybe just pass `onSubmit / onSubmit()`?

Reply:
done